frontend/src/App.jsx

```diff
@@ -85,10 +87,18 @@ export default function App() {
                             </AuthorisedPageShell>
                         }
                     />
+                    <Route
+                        path='/cart' // Adding the cart page route
+                        element={
+                            <AuthorisedPageShell>
+                                <CartPage />
+                            </AuthorisedPageShell>
+                        }
+                     />
```

Comment:
try something similar to the catalogue page I made, then anyone can use the cart

```suggestion
                    <Route
                        path='/cart' // Adding the cart page route
                        element={
                            <AnonPageShell>
                                <CartPage />
                            </AnonPageShell>
                        }
                     />
```